vspace/src/main/java/edu/asu/diging/vspace/core/services/IModuleLinkManager.java

```diff
@@ -0,0 +1,5 @@
+package edu.asu.diging.vspace.core.services;
+
+public interface IModuleLinkManager<IModuleLink,IModule,IModuleLinkDisplay> extends ILinkManager<IModuleLink,IModule,IModuleLinkDisplay>{
```

Comment:
I don't think you need the parameters for the IModuleLinkManager (does Eclipse not show you a warning?)>